### ISSUE TITLE
Revisit default for `editor.suggestSelection` setting

### ISSUE THREAD

**Issue by ajaygera** [microsoft/vscode#136864]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:1.51.1
- OS Version: Windows 10 Pro

To support intellisense in our VSCode extension, I have implemented vscode.CompletionItemProvider interface and call to provideCompletionItems() returns the list of vscode.CompletionList.
When completion item list, gets displayed then scroll bar is not positioned at the first item in the list.
It exists somewhere in the middle of list. To go to the first item I need to manually move scroll bar to the top which is not a good user experience.
As per reply from forum,  this can be controlled by editor.suggestSelection setting.
But Most users of Our VSCode extension, won't be aware of editor.suggestSelection setting
so Is there any other way to keep scroll bar position at first item?
Basically can we set some variable or pass some parameter to VSCode API, to keep scroll bar position at first item?
I have tried setting CompletionItem. isIncomplete to true/false but no luck.

Keeping scroll bar at first position,  is one of the most important requirement for our extension.

Please let me know what I need to do so that scroll bar is always positioned at the first item in the list?

**Comment by jrieken** [comment]
This is controlled via the following user setting: `editor.suggestSelection`. The default is to remember the last selected suggestion (based on context) but I assume your desire is `"editor.suggestSelection": "first"`.

**Comment by jrieken** [comment]
Let's try this during this dev month (for VS Code Insiders) and see what feedback comes in

**Comment by jrieken** [comment]
The default is `first` since a while and I no complaints have been heard. Let's ship this to stable and wait for feedback
